lib/RudderOption.dart

```diff
@@ -4,6 +4,7 @@ import './RudderIntegration.dart';
 // we left fetching the external ids from the scratch here
 class RudderOption {
   List<Map<String, dynamic>>? externalIds;
+  late Map<String, Object> integrations;
```

Comment:
`integrations` object can be nullable if we don't add any integrations. So please implement null-check for integrations object